quickcomm/models.py

```diff
@@ -30,16 +175,60 @@ class Author(models.Model):
     # Taken on: Feb 6, 2023
 
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    host = models.URLField(validators=[URLValidator])
+    # FIXME the user is only null if the author is remote. This should be
+    # enforced in the model.
+    host = models.ForeignKey(Host, on_delete=models.CASCADE, null=True, blank=True,help_text="The host that this author is associated with. This is only used for remote authors. Internal authors will have this field set to null. This field should NOT be set manually.")
+    external_url = models.CharField(max_length=100, blank=True, null=True, help_text="The external URL of the author. This is only used for remote authors. Internal authors will have this field set to null. This field should NOT be set manually.", verbose_name="External URL")
     display_name = models.CharField(max_length=100)
-    github = models.URLField(blank=True, null=True, validators=[URLValidator])
+    github = models.URLField(blank=True, null=True, validators=[URLValidator], help_text="The URL of the author's GitHub profile. This must be in proper form (e.g. https://github.com/abramhindle).", verbose_name="GitHub URL")
```

Comment:
did you make a change that requires a full url? when I was testing github things locally from the main branch I needed to use my nickname, the URL was giving me the error